freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncService.java

```diff
@@ -276,18 +278,26 @@ private SyncStatusDetail internalSynchronizeStack(Stack stack, UmsUsersState ums
         Multimap<String, String> warnings = ArrayListMultimap.create();
         try {
             FreeIpaClient freeIpaClient = freeIpaClientFactory.getFreeIpaClientForStack(stack);
+            LOGGER.debug("Starting RETRIEVE_{}_IPA_STATE ...", fullSync ? "FULL" : "PARTIAL");
             UsersState ipaUsersState = getIpaUserState(freeIpaClient, umsUsersState, fullSync);
-            LOGGER.debug("IPA UsersState, found {} users and {} groups", ipaUsersState.getUsers().size(), ipaUsersState.getGroups().size());
+            LOGGER.debug("Finished RETRIEVE_{}_IPA_STATE, found {} users and {} groups.", fullSync ? "FULL" : "PARTIAL",
```

Comment:
this `FULL` vs `PARTIAL` is duplicated. maybe it should be move to a variable or a method